powertools-core/src/main/java/software/amazon/lambda/powertools/core/internal/LambdaHandlerProcessor.java

```diff
@@ -47,6 +47,13 @@ public static boolean isHandlerMethod(final ProceedingJoinPoint pjp) {
         return placedOnRequestHandler(pjp) || placedOnStreamHandler(pjp);
     }
 
+    /**
```

Comment:
why adding this?

Reply:
Because it took me 15 minutes to understand what we're doing here.
May save someone that time